app/scripts/modules/core/modal/wizard/v2wizardPage.directive.js

```diff
@@ -38,6 +38,7 @@ module.exports = angular
         $scope.label = attrs.label;
         $scope.state = state;
         v2modalWizardService.registerPage($scope.key, $scope.label, state);
+        $scope.$on('$destroy', () => v2modalWizardService.setRendered($scope.key, false));
```

Comment:
One change from @anotherchrisberry to delete the modal wizard tabs correctly. Ready to merge?